src/app.js

```diff
@@ -0,0 +1,29 @@
+require("dotenv").config();
+const express = require('express');
+const connectDB = require('./database');
+const errorHandler = require("./middlewares/errorHandler.middleware");
+const cartRouter = require("./routes/cart.route");
+const productRouter = require("./routes/products.route");
+const userRouter = require("./routes/user.route");
+const wishlistRouter = require("./routes/wishlist.route");
+const cors = require('cors');
+const app = express()
+const port = process.env.PORT || 3000;
+
```

Comment:
I have seen some folks giving global variable upper case names, when it is immutable, like PORT and not "port".